realsense2_camera/src/profile_manager.cpp

```diff
@@ -88,7 +88,14 @@ void ProfilesManager::registerSensorUpdateParam(std::string template_name,
     {
         std::string param_name = applyTemplateName(template_name, sip);
         if (params.find(sip) == params.end())
-            params[sip] = std::make_shared<T>(value);
```

Comment:
Here, by default the "_enable__*" param will be set to 'true' for all video streams (Depth, Infra0,1&2).